test/Serializers/Tests/Phpunit/DispatchingSerializerTest.php

```diff
@@ -27,7 +27,7 @@ public function testConstructWithNoSerializers() {
 
 	public function testConstructWithInvalidArgumentsCausesException() {
 		$this->setExpectedException( 'InvalidArgumentException' );
-		new DispatchingSerializer( array( new \stdClass() ) );
+		new DispatchingSerializer( [ new \stdClass() ] );
```

Comment:
I would like to see this using "use stdClass". But that's something for a later PR.